.circleci/config.yml

```diff
@@ -0,0 +1,34 @@
+version: 2.0
+jobs:
+  build:
+    docker:
+      - image: grpc/python
+
+    steps:
+      - checkout
+      - run:
+          name: Install deps
+          command: pip install --no-cache-dir -r ./examples/python/requirements.txt
+
+      - run:
+          name: Start FullScriptSupport plugin
+          command: python ./examples/python/FullScriptSupport/ExtensionService_Script.py # :50051
+          background: true
+
+      - run:
+          name: Start HelloWorld plugin
+          command: python ./examples/python/HelloWorld/ExtensionService_helloworld.py  # :50052
```

Comment:
Just a "philosophical" thought, why is the helloworld example the second to be started? And then running the tests as test_* actually makes the helloworld tests run last because of the alphabetical sorting of the test files test_*. Nitpicking, but would be nice to see the same logical order starting the examples and then running their tests, maybe on a separate PR.